[2024-11-03T02:02:15.790+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T02:02:15.799+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_sales_data_to_db.fetch_sales_data_raw scheduled__2024-11-03T01:27:00+00:00 [queued]>
[2024-11-03T02:02:15.806+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_sales_data_to_db.fetch_sales_data_raw scheduled__2024-11-03T01:27:00+00:00 [queued]>
[2024-11-03T02:02:15.807+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-03T02:02:15.820+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fetch_sales_data_raw> on 2024-11-03 01:27:00+00:00
[2024-11-03T02:02:15.825+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3093) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T02:02:15.827+0000] {standard_task_runner.py:72} INFO - Started process 3103 to run task
[2024-11-03T02:02:15.828+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_sales_data_to_db', 'fetch_sales_data_raw', 'scheduled__2024-11-03T01:27:00+00:00', '--job-id', '321', '--raw', '--subdir', 'DAGS_FOLDER/raw/sales.py', '--cfg-path', '/tmp/tmpi3y1sz3i']
[2024-11-03T02:02:15.828+0000] {standard_task_runner.py:105} INFO - Job 321: Subtask fetch_sales_data_raw
[2024-11-03T02:02:15.876+0000] {task_command.py:467} INFO - Running <TaskInstance: load_sales_data_to_db.fetch_sales_data_raw scheduled__2024-11-03T01:27:00+00:00 [running]> on host f05559d9e656
[2024-11-03T02:02:15.950+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_sales_data_to_db' AIRFLOW_CTX_TASK_ID='fetch_sales_data_raw' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T01:27:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-03T01:27:00+00:00'
[2024-11-03T02:02:15.951+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-03T02:02:15.951+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-03T02:02:15.951+0000] {logging_mixin.py:190} INFO - Current task name:fetch_sales_data_raw state:running start_date:2024-11-03 02:02:15.800035+00:00
[2024-11-03T02:02:15.951+0000] {logging_mixin.py:190} INFO - Dag name:load_sales_data_to_db and current dag run status:running
[2024-11-03T02:02:15.952+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T02:22:15.617+0000] {job.py:229} INFO - Heartbeat recovered after 1194.77 seconds
[2024-11-03T02:35:04.742+0000] {job.py:229} INFO - Heartbeat recovered after 754.04 seconds
